### ISSUE TITLE
Re-arrange order of Dice value entry

### ISSUE THREAD

**Issue by vjcortesa** [averbraeck/housinggame-facilitator#27]
- [ ] The dice value entry is prone to errors: In other places of the facilitator app and the board map, the river and rain dice value and protection level is always in place in that order, except for the dice value entry. As a result, the facilitator has to read various times when putting the value in. To keep it consistent, and avoid errors, rearrange accordingly following the player order, first rain and then river everywhere.
- [ ] The technical terms Fluvial and Pluvial are only used in the dice entry: To keep it consistent, rephrase it to the game-used terms river and rain.
![240224_Invert the order of the dice value entry](https://github.com/averbraeck/housinggame-facilitator/assets/8215750/201d0860-8cbc-49a0-b556-d8c3c60bb328)

**Comment by averbraeck** [comment]
To make everything consistent, first river, then rain is used. 
The terms fluvial and pluvial will be removed everywhere, except for the admin app.

**Comment by averbraeck** [comment]
Done:

![image](https://github.com/averbraeck/housinggame-facilitator/assets/13779793/6b0212fc-2378-4253-b263-c169d81c3d12)
